### ISSUE TITLE
Discernible focus state needed on  "What's new? post in" select on News Feed

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/knowledge-commons-profiles#381]
Area of site: News Feed
URL: https://hcommons-staging.org/activity/

**Description**
The select dropdown (`<select id="whats-new-post-in" name="whats-new-post-in">`) that appears when the "What's new, ___?" is focused does not have a discernible focus state when it is focused. This means, for example, users who are navigating to this element with a keyboard are unable to determine if this element is focused and ready to be interacted with. 

**Recommendation**
Remove or rewrite the `opacity: 0;` rule that appears on line 609 in `main-global.css` as it applies to this element.

```
.item-list-tabs select, #whats-new-form select, .editfield select:not([multiple]), #notifications-bulk-management select, #messages-bulk-management select, .field-visibility select, .register-section select, .bbp-form select, #bp-group-course, #bbp_group_forum_id {
    opacity: 0;
}
```
